portfolio/src/main/webapp/script.js

```diff
@@ -65,22 +67,36 @@ async function getRandomProverb(){
 }
 
 function printComments() {
-  
     // Fetch comments from the server
     fetch('/data?max='+document.getElementById('quantity').value).then(response => response.json()).then((comment) => {
       const commentElements = document.getElementById('user-message');
-      // Runs a loop for each content of the comments array to be printed out
-      comment.forEach((comments) => {
-        commentElements.appendChild(createListElement(comments));
-      })
+
+    // Runs a loop for each content of the comments array to be printed out
+      for (var i = 0; i < comment.comments_.length; i++) {
+          //commentElements.appendChild(createheadElement(comment.comments_[i]));
+          commentElements.appendChild(createListElement(comment.comments_[i]));  
+        }
 });
 }
 
 /** Creates an <p> element containing text. */
 function createListElement(text) {
+  const liElem = document.createElement('li');
+
+  const heading = document.createElement('h5');
+  if(String.fromCharCode.apply(String, text.name_.bytes) === ""){
```

Comment:
I am not sure if there a Google style formatter for JavaScript, but I would suggest leaving a blank space between 'if' and '('.  Similarly, leave a blank space between ')' and '{'.

Reply:
Corrected!